public/js/controllers/address_controller.js

```diff
@@ -175,21 +175,35 @@ export default class extends Controller {
     ctrl.bindElements()
     ctrl.bindEvents()
     ctrl.query = new TurboQuery()
-
+    ctrl.defaultSettings = {
+      chart: 'balance',
+      zoom: 'ijhhasg0-kbhgjkw0',
+      zoombutton: 'all',
+      bin: 'month',
+      flow: '',
+      n: '',
+      start: '',
+      txntype: ''
+    }
+    // ?chart=balance&zoom=ijhhasg0-kbhgjkw0&bin=month
     // These two are templates for query parameter sets.
     // When url query parameters are set, these will also be updated.
     var settings = ctrl.settings = TurboQuery.nullTemplate(['chart', 'zoom', 'bin', 'flow',
-      'n', 'start', 'txntype'])
+      'n', 'start', 'txntype', 'zoombutton'])
```

Comment:
Since we have `zoom` in the settings, it looks like `zoombutton` is now a repetition

Reply:
Resolved.